app/services/analytics_events.rb

```diff
@@ -3737,14 +3737,23 @@ def password_max_attempts
   # confirmed email
   # @param [Boolean, nil] active_profile if the account the reset is being requested for has an
   # active proofed profile
+  # @param [Hash] error_details Details for error that occurred in unsuccessful submission
   # The user entered an email address to request a password reset
-  def password_reset_email(success:, errors:, confirmed:, active_profile:, **extra)
+  def password_reset_email(
+    success:,
+    errors:,
+    confirmed:,
+    active_profile:,
+    error_details: {},
```

Comment:
I'm wondering why we (read: I) made it so that `error_details` is only assigned to a `FormResponse` when non-empty:

https://github.com/18F/identity-idp/blob/c5e5d8a06237a7913fb71131aa397ce7e4fd8f88/app/services/form_response.rb#L21

It'd make things a lot simpler if it were just always there, as an empty hash if it were empty.

Reply:
maybe to minimize impact in the cases where we `.compact`